doc/source/tutorials.rst

```diff
@@ -125,6 +125,31 @@ There are four sections covering selected topics as follows:
 
 .. _tutorial-modern:
 
+Exercises for New Users
+-----------------------------------
```

Comment:
line needs to be the same length as the title.

add a ref-tag (e.g. like _tutorial-exercises-new-users) and move the modern one down.